media-libs/libtiger/libtiger-0.3.4-r1.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit libtool multilib-minimal
+
+DESCRIPTION="A rendering library for Kate streams using Pango and Cairo"
+HOMEPAGE="https://code.google.com/p/libtiger/"
+SRC_URI="https://libtiger.googlecode.com/files/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-fbsd ~x86-fbsd"
+IUSE="doc"
+
+RDEPEND="x11-libs/pango[${MULTILIB_USEDEP}]
+	>=media-libs/libkate-0.2.0[${MULTILIB_USEDEP}]
+	x11-libs/cairo[${MULTILIB_USEDEP}]"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+	doc? ( app-doc/doxygen )"
+
+src_prepare() {
+	eapply_user
```

Comment:
Lemme see… because it was the first thing that came to my head? ;-P Switched to `default`.